test/core/client_channel/lb_policy/BUILD

```diff
@@ -167,6 +174,7 @@ grpc_cc_test(
         "gtest",
     ],
     language = "C++",
+    tags = ["lb_unit_test"],
```

Comment:
This test doesn't need this tag, since it's only covering the scheduler, not the LB policy itself.

Reply:
done